contribute-to-tidb/code-style-and-quality-guide.md

```diff
@@ -38,3 +38,20 @@ func main() {
     fmt.Println(ping.Domain())
 }
 ```
+
+## Consider using values instead of pointers
+
+Many Go programmers over-use pointers as opposed to just using values directly.
+Go programs often return pointers to structs with a wrapper and then all functions use the struct as a pointer:
```

Comment:
```suggestion
Many Go programmers over-use pointers as opposed to just using values directly. Go programs often return pointers to structs with a wrapper and then all functions use the struct as a pointer:
```

Personal style preference. If you have different opinion, please just resolve this suggestion.